frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -565,7 +565,7 @@ function ReaderFooter:addToMainMenu(menu_items)
             {
                 text = _("Alignment"),
                 enabled_func = function()
-                    return self.settings.disable_progress_bar
+                    return not self.settings.disable_progress_bar == false
                 end,
```

Comment:
Maybe some parentheses for clarity?

Reply:
Or just `return self.settings.disable_progress_bar == true`.
(our touchmenu code considers `nil` to mean: enabled - it just checks for `false` meaning: disabled)